napari_aicssegmentation/core/router.py

```diff
@@ -22,6 +22,6 @@ def workflow_selection(self):
         self._controller = WorkflowSelectController(self._application, self._layer_reader, self._workflow_engine)
         self._controller.index()
 
-    def workflow_steps(self):
-        self._controller = WorkflowStepsController(self._application, self._workflow_engine)
+    def workflow_steps(self, selected_workflow: str):
```

Comment:
Same as above - we have the selected workflow in the model so no need to worry about passing it around here.